src/main/java/com/werfen/report/service/pdf/PdfGridReportService.java

```diff
@@ -14,22 +14,29 @@
 import net.sf.jasperreports.engine.JasperPrint;
 import net.sf.jasperreports.engine.design.JasperDesign;
 
+import java.io.ByteArrayOutputStream;
+
 public class PdfGridReportService {
 
     public void export(String filePath, GridReportConfiguration gridReportConfiguration, GridPageDataSource gridPageDataSource, PageFormat pageFormat) throws JRException {
```

Comment:
In the same line as other comments, I would rename this method to be exportToFile

Reply:
Agree, it will be changed in the following commit.